src/community/notification/src/main/java/org/geoserver/notification/NotificationCatalogListener.java

```diff
@@ -1,266 +1,270 @@
-                                            transformer)
-                                    .toArray());
-            String newStyles =
-                    StringUtils.join(
-                            CollectionUtils.collect(
-                                            (Set<StyleInfo>) newValues.get(stylesIndex),
-                                            transformer)
-                                    .toArray());
-            if (!oldStyles.equals(newStyles)) {
-                properties.put("styles", newStyles);
-            }
-        }
-    }
-
-    private void handleWorkspaceRename(
-            Map<String, Object> properties,
-            final CatalogInfo source,
-            final List<String> changedProperties,
-            final List<Object> oldValues,
-            final List<Object> newValues) {
-        final int nameIndex = changedProperties.indexOf("name");
-        final String oldWorkspaceName = (String) oldValues.get(nameIndex);
-        final String newWorkspaceName = (String) newValues.get(nameIndex);
-    }
-
-    private void handleRename(
-            Map<String, Object> properties,
-            final CatalogInfo source,
-            final List<String> changedProperties,
-            final List<Object> oldValues,
-            final List<Object> newValues) {
-
-        final int nameIndex = changedProperties.indexOf("name");
-        final int namespaceIndex = changedProperties.indexOf("namespace");
-
-        String oldLayerName;
-        String newLayerName;
-        if (source instanceof ResourceInfo) { // covers LayerInfo, CoverageInfo, and WMSLayerInfo
-            // must cover prefix:name
-            final ResourceInfo resourceInfo = (ResourceInfo) source;
-            final NamespaceInfo currNamespace = resourceInfo.getNamespace();
-            final NamespaceInfo oldNamespace;
-            if (namespaceIndex > -1) {
-                oldNamespace = (NamespaceInfo) oldValues.get(namespaceIndex);
-            } else {
-                oldNamespace = currNamespace;
-            }
-
-            newLayerName = resourceInfo.prefixedName();
-            if (nameIndex > -1) {
-                oldLayerName = (String) oldValues.get(nameIndex);
-            } else {
-                oldLayerName = resourceInfo.getName();
-            }
-            oldLayerName = oldNamespace.getPrefix() + ":" + oldLayerName;
-        }
-    }
-
-    @Override
-    public void setMessageMultiplexer(MessageMultiplexer messageMultiplexer) {
-        this.messageMultiplexer = messageMultiplexer;
-    }
-
-    @Override
-    public MessageMultiplexer getMessageMultiplexer() {
-        return messageMultiplexer;
-    }
-
-    @Override
-    public void handlePostModifyEvent(CatalogPostModifyEvent event) throws CatalogException {
-        int a = 1;
-    }
-
-    @Override
-    public void reloaded() {}
-}
+/* (c) 2017 Open Source Geospatial Foundation - all rights reserved
```

Comment:
Impossible to understand what ha been changed, please run maven format or at least add google format [plugin to your IDE](https://docs.geoserver.org/stable/en/developer/eclipse-guide/index.html#code-formatting).

Reply:
thats eclipse doing its thing, i will revert this class